src/pages/AddLiquidity/index.tsx

```diff
@@ -434,11 +433,6 @@ function AddLiquidity() {
     const maxPrice = pricesAtLimit[Bound.UPPER]
     if (maxPrice) searchParams.set('maxPrice', maxPrice.toSignificant(5))
     setSearchParams(searchParams)
-
-    sendEvent({
-      category: 'Liquidity',
-      action: 'Full Range Clicked',
-    })
```

Comment:
are we no longer tracking this one?

Reply:
no i don't think it's necessary but we should be including more details in the add events... updating

Reply:
also this wasn't on the list of events Blair requested we migrate in #6554